src/components/molecules/LoadingPage/loading.scss

```diff
@@ -1,137 +1,178 @@
-  &.no-scroll {
-    overflow: hidden;
-  }
-}
+$loadingburningman-duration: $sparkleanim-duration/2;
+$loadingburningman-opacity-min: 0;
+$loadingburningman-opacity-max: 1;
+$loadingburningman-bg-color: $white;
+
+.LoadingPage {
+  // Setting position here makes our container the 'nearest positioned ancestor' when positioning child elements
+  position: absolute;
+
+  width: 100vw;
+  height: 100vh;
 
-.loading-screen {
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
   background-color: $primary;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  transition: opacity 1000ms cubic-bezier(0.23, 1, 0.32, 1);
-  opacity: 1;
-  &.hide {
-    opacity: 0;
+  background-image: $loading-page-background-image;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &__blur {
+    width: inherit;
+    height: inherit;
+
+    // Gives us the blurred effect over the background image (when set)
+    background-color: transparent;
+    backdrop-filter: blur(15px);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
   }
 
-  .burningman-loading-container {
-    position: relative;
+  &__image {
+    // CSS ignores invalid property values
+    //   When this var is set to an image URL, the browser will ignore it
+    //   When this var isn't set, then we will use the default fallback for the var, which is 'none'
+    display: $loading-page-background-image;
+
+    border-radius: 20px;
+
+    // Background image size / aspect ratio hack (https://css-tricks.com/aspect-ratio-boxes/)
+    // @debt We can probably replace this with aspect-ratio when it's supported across more browsers
+    width: 30%;
+    height: 0;
+    padding-top: calc(800 / 600 * 30%);
+
+    // Fallback to showing the main container's background in case we don't load an image
+    background-color: transparent;
+
+    background-image: $loading-page-background-image;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
   }
-  .loading-sparkle-1,
-  .loading-sparkle-2 {
+
+  &__loading-text {
     position: absolute;
-    display: block;
-    z-index: z(loading-page-sparkle);
-    height: 23px;
-    width: 15px;
-    opacity: 1;
-    left: 20%;
-    transform: scale(1);
-    background-image: url(./img/loading-sparkle-icon.png);
-    background-size: 15px 23px;
-    animation: pulse 2s cubic-bezier(0.77, 0, 0.175, 1) infinite;
-    opacity: 0;
-    &.loading-sparkle-2 {
-      right: 16%;
-      left: auto;
-      top: 20%;
-      animation-delay: 0.2s;
-    }
+    bottom: 0;
+
+    font-family: $kotr-font-larger;
+    font-size: $loading-text-size;
+
+    color: $white;
+    text-shadow: 5px 5px 20px rgba($black, 1);
+
+    user-select: none;
   }
+}
+
+// @debt extract this loading icon into a standalone component
+.loading-icon {
+  position: absolute;
+  bottom: 1vw;
+  left: calc(50vw + 18vw);
```

Comment:
Did you test it on different screens?

Reply:
Tested on my main screen + laptop and dragged it around a bit in between. Seemed to look decent enough. The middle image could potentially get larger on a mobile view if we wanted to get pedantic about it, but that's about the biggest 'issue' I found.